.github/workflows/auto-merge.yml

```
@@ -0,0 +1,10 @@
+name: "auto-merge"
+on: [pull_request_target]
+
+jobs:
+  auto-merge:
+    uses: mdn/workflows/.github/workflows/auto-merge.yml@main
+    with:
+      target-repo: "TODO: add the name of the current (origin) repo"
```

Should we do "e.g., `'mdn/content'` or something?

```suggestion
      target-repo: "TODO: set this value to the name of the current (origin) repo"
```